rest_api/rest_api/controller/feedback.py

```diff
@@ -113,7 +113,7 @@ def export_feedback(
 
         offset_start_in_document = 0
         if label.answer and label.answer.offsets_in_document:
-            offset_start_in_document = label.answer.offsets_in_document[0].start
+            offset_start_in_document = label.answer.offsets_in_document[0].row
```

Comment:
mypy was failing, so I corrected this line according to what the TableCell proposal specifies: 

https://github.com/deepset-ai/haystack/blob/1be39367ac6e17fa7f38d228c86ccc1996c57108/proposals/text/3875-table-cell.md?plain=1#L210